test/unit_tests/25_host_multirate/aieSingleOutBuffer.mlir

```diff
@@ -9,28 +9,16 @@
 //===----------------------------------------------------------------------===//
 
 // REQUIRES: valid_xchess_license
-// RUN: aiecc.py -j4 --sysroot=%VITIS_SYSROOT% --host-target=aarch64-linux-gnu %s -I%aie_runtime_lib%/ %extraAieCcFlags% %aie_runtime_lib%/test_library.cpp %S/test.cpp -o test.elf
+// RUN: aiecc.py -j4 --sysroot=%VITIS_SYSROOT% --host-target=aarch64-linux-gnu %s -I%aie_runtime_lib%/ %extraAieCcFlags% %aie_runtime_lib%/test_library.cpp %S/testSingleOutBuffer.cpp -o host_multirate_SB.exe
 // RUN: %run_on_board ./test.elf
 
 module @host_multirate {
     %tile34 = AIE.tile(3, 4)
     %tile70 = AIE.tile(7, 0)
 
-    %hostLock = AIE.lock(%tile34, 0) {sym_name="hostLock"}
-
-    func.func @evaluate_condition(%argIn : i32) -> (i1) {
-        %true = arith.constant 1 : i1
-        return %true : i1
-    }
-
-    func.func @payload(%argIn : i32) -> (i32) {
-        %next = arith.constant 1 : i32
-        return %next : i32
-    }
-
     %ext_buf70_in  = AIE.external_buffer {sym_name = "ddr_test_buffer_in"}: memref<256xi32> 
     %ext_buf70_out = AIE.external_buffer {sym_name = "ddr_test_buffer_out"}: memref<64xi32> 
-
+   
```

Comment:
We should have some sensible project-wide editor configuration to avoid this random space additions.